docs/UserGuide.md

```diff
@@ -86,7 +86,7 @@ Format: `help`
 
 Adds a person to uNivUSal.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​`
+Format: `add o/OCCUPATION n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​`
```

Comment:
> Conflict between the Userguide.md, think you should have removed the extra field to resolve this.

Yes, specifically, it's missing `a/ADDRESS` here.

Reply:
Oh I see it now, thanks for the advice!